cluster/aws/util.sh

```diff
@@ -1211,7 +1211,13 @@ function wait-master() {
   echo "  up."
   echo
 
-  until $(curl --insecure --user ${KUBE_USER}:${KUBE_PASSWORD} --max-time 5 \
+  if [[ $(curl -V | awk '{print $2}' | head -1) > 7.34.0 ]]; then
+    TLS_OPTS="--insecure --tlsv1.2"
```

Comment:
Mike,

`--insecure` tells libcurl to not verify the peer (https://curl.haxx.se/docs/sslcerts.html). If you see the man page, it has additional details

```
(SSL) This option explicitly allows curl to perform "insecure" SSL connections and transfers. All SSL connections are attempted to be made secure by using the CA certificate bundle installed by default. This makes all connections considered "insecure" fail unless -k, --insecure is used.
```
fyi. bug reporter has confirmed that adding this option makes the scenario work.

the bug reporter has confirmed that adding --tlsv1.2 works as well. 

Thanks,
Dims